debian/pbspro-execution.prerm

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+# postrm script for pbspro
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postrm> `remove'
+#        * <postrm> `purge'
+#        * <old-postrm> `upgrade' <new-version>
+#        * <new-postrm> `failed-upgrade' <old-version>
+#        * <new-postrm> `abort-install'
+#        * <new-postrm> `abort-install' <old-version>
+#        * <new-postrm> `abort-upgrade' <old-version>
+#        * <disappearer's-postrm> `disappear' <overwriter>
+#          <overwriter-version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+
+case "$1" in
+    purge|remove|upgrade|failed-upgrade|abort-install|abort-upgrade|disappear)
+    if [ -x /sbin/chkconfig -a -x /sbin/service ]; then
+        out=`/sbin/chkconfig --list pbs 2>/dev/null`
+        if [ $? -eq 0 ]; then
+            if [ -x /etc/init.d/pbs ]; then
+                    /etc/init.d/pbs stop
+            else
+                    /sbin/service pbs stop
+            fi
```

Comment:
@subhasisb, @mike0042, I would suggest that we concentrate on Ubuntu/Debian related changes in this pull request. For changes to pbspro.spec, we can create another ticket.